### PR TITLE
修改绑定问题，原来的写法报错误Illegal offset type in unset

### PR DESCRIPTION
**Pull request by zigo928**
原来的写法报错误Illegal offset type in unset

### CODE REVIEW

**Comment by liyu001989** [comment]
这么写是为了 app('Liyu\Signature\SignatureManager');  这样也能正确使用。

你是laravel 什么版本，报错也粘贴一下吧

我本地正常的

**Comment by zigo928** [comment]
我是5.4版本，错误提示如下：

[2017-03-31 15:23:24] local.ERROR: ErrorException: Illegal offset type in unset in /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Container/Container.php:1057
Stack trace:
#0 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Container/Container.php(1057): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Illegal offset ...', '/Users/gengzhig...', 1057, Array)
#1 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Container/Container.php(211): Illuminate\Container\Container->dropStaleInstances(Array)
#2 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/liyu/signature/src/ServiceProvider.php(46): Illuminate\Container\Container->bind(Array, Object(Closure))
#3 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(568): Liyu\Signature\ServiceProvider->register()
#4 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php(75): Illuminate\Foundation\Application->register(Object(Liyu\Signature\ServiceProvider))
#5 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(543): Illuminate\Foundation\ProviderRepository->load(Array)
#6 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#7 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(208): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#8 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(270): Illuminate\Foundation\Application->bootstrapWith(Array)
#9 /Users/gengzhiguo/luosidao/lsd-merchant/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(115): Illuminate\Foundation\Console\Kernel->bootstrap()
#10 /Users/gengzhiguo/luosidao/lsd-merchant/artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 {main}

**Comment by zigo928** [comment]
应该是bind函数第一个参数为数组是有问题的

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "liyu/signature",
+    "name": "liuyu/signature",
```
Comment (liyu001989) [inline composer.json:2]: 你手误了吧..包名..   5.4 确实有问题 ，稍等我测试一下

**Comment by liyu001989** [comment]
感谢，应该是 https://github.com/liyu001989/signature/commit/2d414c4cdb9cfdd2f471d8d0e9e5285807a38ed3
这样修改，composer update 试一下

**Pull request closed without merging**